simapp/app.go

```diff
@@ -76,7 +76,7 @@ const appName = "SimApp"
 
 var (
 	// DefaultCLIHome default home directories for the application CLI
-	DefaultCLIHome = os.ExpandEnv("$HOME/.simapp")
+	DefaultCLIHome = os.ExpandEnv("$HOME/.simappcli")
```

Comment:
I do believe this was meant to stay as simapp. Is there a reason it needs to be changed?

Reply:
Is there a reason the cli and the daemon share the same directory?

I think this causes an issue, for example overwriting the config with two different formats.

Reply:
I can undo this and we just rely on the docker mounts to use different folders.

Reply:
I never understood this change and the reasoning behind it. maybe @alexanderbez or @alessio could shed some light?